mkdocs.yml

```diff
@@ -289,6 +289,8 @@ pages:
       - Overview: agents/index.md
       - Sematext Agent:
         - Overview: agents/sematext-agent/index.md
+        - Release Notes: agents/sematext-agent/platform-support-policy.md
```

Comment:
Release Notes ->  Platform Support Policy

cc @otisg that reported this (we'll work the platform support policy on this PR, the previous has been reverted)